pkg/platform/storage/git/application.go

```diff
@@ -136,6 +126,7 @@ func (s *GitStorage) GetApplications(tenantID string) ([]platform.HttpResponseAp
 	for _, applicationID := range applicationIDs {
 		application, err := s.GetApplication(tenantID, applicationID)
 		if err != nil {
+			fmt.Println(err)
```

Comment:
Maybe nice with a little context for this error:
```suggestion
			fmt.Println("Skipping application for tenant", tenantID, "with ID", applicationID, "because it failed to load:", err)
```